### PR TITLE
add Standard & initiate the DevOps category

### PR DESCRIPTION
**Pull request by blaggacao**
makes is the inspirational predecessor of std, so it belongs here,too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,14 +112,18 @@
...
 * [nix-direnv](https://github.com/nix-community/nix-direnv) - A fast loader and flake-compliant configuration for the direnv environment auto-loader.
 * [nixpkgs-review](https://github.com/Mic92/nixpkgs-review) - The best tool to verify that a pull-request in Nixpkgs is building properly.
 * [pre-commit-hooks.nix](https://github.com/cachix/pre-commit-hooks.nix) - Run linters/formatters at commit time and on your CI.
 * [rnix-lsp](https://github.com/nix-community/rnix-lsp) - A syntax-checking language server for Nix.
 * [robotnix](https://github.com/danielfullmer/robotnix) - A declarative and reproducible build system for Android (AOSP) images.
 
+## DevOps
+
+* [Makes](https://github.com/fluidattacks/makes) - A Nix-based CI/CD pipeline framework for building, testing, and releasing projects in any language, from anywhere. 
+* [Standard](https://github.com/divnix/std) - The Nix Flakes DevOps framework for perfectionists with deadlines
```
Comment (houstdav000) [inline README.md:125]: The description is extremely vague. It does use Nix-Flakes, and is a framework, but might be better put as "An opinionated Nix Flakes development framework to organize large projects with folder naming.".

That could probably be better-worded, but it's definitely more descriptive.

Also, make sure to add proper punctuation to get it to pass the `awesome-lint` action.

File: README.md [hunk 2]
```diff
@@ -112,14 +112,18 @@
...
 * [lorri](https://github.com/nix-community/lorri/) - A much better `nix-shell` for development that augments direnv.
-* [Makes](https://github.com/fluidattacks/makes) - A Nix-based CI/CD pipeline framework for building, testing, and releasing projects in any language, from anywhere. 
 * [niv](https://github.com/nmattia/niv/) - Easy dependency management for Nix projects with package pinning.
 * [nix-direnv](https://github.com/nix-community/nix-direnv) - A fast loader and flake-compliant configuration for the direnv environment auto-loader.
 * [nixpkgs-review](https://github.com/Mic92/nixpkgs-review) - The best tool to verify that a pull-request in Nixpkgs is building properly.
 * [pre-commit-hooks.nix](https://github.com/cachix/pre-commit-hooks.nix) - Run linters/formatters at commit time and on your CI.
 * [rnix-lsp](https://github.com/nix-community/rnix-lsp) - A syntax-checking language server for Nix.
 * [robotnix](https://github.com/danielfullmer/robotnix) - A declarative and reproducible build system for Android (AOSP) images.
 
+## DevOps
```
Comment (houstdav000) [inline README.md:122]: If we're going to create a new section, be sure to add it into the Table of Contents at the top.

**Comment by blaggacao** [comment]
Thanks for the friendly review!

I've tended to your feedback. Can you revise again?

Thank you!

**Review by houstdav000 — APPROVED** [review]
Looks good now

**Pull request merged**